### PR TITLE
Remove Travis from docs

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR

- Update documentation to replace Travis-specific instructions with those required when using GitHub Actions

We switched from Travis to GitHub Actions to run our deployments.

Most of the documentation was updated around that time [1], but there were still mentions of Travis here and there.

I'm mostly guessing at the env var steps, so that's where I'd like a review / confirmation!

## Screenshots of UI changes

N/A

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [ ] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

File: doc/manage-environment-variables.md
Comment (leeky) [inline doc/manage-environment-variables.md:27]: Those `PROD_` and `STAGING_` prefixes aren't magic, so it looks like any new environment variables also need to be referenced in these files too:

- `.github/workflows/deploy.yml`
- `terraform/variables.tf`
- `terraform/app.tf`

Comment (CristinaRO) [reply doc/manage-environment-variables.md:27]: Ah, thanks, the previous instructions made it seem like the prefixes were indeed magic :O

**Comment by pezholio** [comment]
@CristinaRO What's blocking this from being merged?

**Comment by CristinaRO** [comment]
> @CristinaRO What's blocking this from being merged?

I've run out of time and brain to address Leeky's comment. (And there were other priorities.)

**Comment by mec** [comment]
I will do my best to finish this up…

**Comment by mec** [comment]
@leeky I think I've addressed all the suggestions you made - would you mind taking a look?

File: doc/manage-environment-variables.md
Comment (leeky) [reply doc/manage-environment-variables.md:34]: @mec I think you still need to add something about adding the environment variables to `terraform/variables.tf` (?)

**Review by leeky — APPROVED** [review]
I think there's still a step missing from the instructions, but otherwise LGTM!

File: doc/manage-environment-variables.md
Comment (mec) [reply doc/manage-environment-variables.md:34]: Sorry, is that not step 1?

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/manage-environment-variables.md

Comment (leeky) [reply doc/manage-environment-variables.md:34]: Ah... it wasn't showing in the diff 🤦

**Pull request merged**
